bigtable/admin.go

```diff
@@ -597,12 +597,18 @@ func (ac *AdminClient) WaitForReplication(ctx context.Context, table string) err
 	}
 }
 
-// TableIAM creates an IAM client specific to a given Instance and Table within the configured project.
+// TableIAM creates an IAM Handle specific to a given Instance and Table within the configured project.
 func (ac *AdminClient) TableIAM(tableID string) *iam.Handle {
 	return iam.InternalNewHandleGRPCClient(ac.tClient,
 		"projects/"+ac.project+"/instances/"+ac.instance+"/tables/"+tableID)
 }
 
+// BackupIAM creates an IAM Handle specific to a given Cluster and Backup.
+func (ac *AdminClient) BackupIAM(ctx context.Context, cluster, backup string) *iam.Handle {
+	backupPath := ac.instancePrefix() + "/clusters/" + cluster + "/backups/" + backup
```

Comment:
is this anywhere else in the code? If not, it seems useful as a common utility.

Reply:
I've met this code in few places. In [UpdateBackup](https://github.com/googleapis/google-cloud-go/blob/2a980a2cbae52faffd778c75767c3f96f121a2d1/bigtable/admin.go#L1642-L1646), [BackupInfo](https://github.com/googleapis/google-cloud-go/blob/2a980a2cbae52faffd778c75767c3f96f121a2d1/bigtable/admin.go#L1604-L1608), [RestoreTable](https://github.com/googleapis/google-cloud-go/blob/2a980a2cbae52faffd778c75767c3f96f121a2d1/bigtable/admin.go#L1443-L1446), [DeleteBackup](https://github.com/googleapis/google-cloud-go/blob/2a980a2cbae52faffd778c75767c3f96f121a2d1/bigtable/admin.go#L1628-L1632)

Reply:
cool - I think it would be worth it to refactor this as a utility helper then in this PR

Reply:
Sorry for the delay here, I had an emergency.

Done, thanks for advice!

Reply:
No worries, lgtm thank you!